packages/components/src/Combobox/components/ComboboxAction/ComboboxAction.css

```diff
@@ -8,15 +8,22 @@
 .actionButton {
   position: relative;
   width: 100%;
+  min-height: calc((var(--space-largest) - var(--space-small)));
   padding: var(--space-small) 0;
   border: none;
   text-align: left;
   background-color: var(--color-surface);
+  cursor: pointer;
 }
 
 .actionButton:focus {
+  box-shadow: var(--shadow-focus);
+  border-radius: var(--radius-large);
```

Comment:
Couple of really minor state things that would be nice to see addressed as a fast-follow 

- is there a reason we're manipulating the border-radius on a per-state basis? As far as I can tell we could simplify this and the hover state on line25 by setting the border-radius on `.actionButton`
- can we set both the options and actions to have a `transition: all var(--timing-base);` applied to match [Button](https://github.com/GetJobber/atlantis/blob/e46abe091488d7bacf653ac8fd7ffb73b575e8aa/packages/components/src/Button/Button.css#L26C38-L26C38)?

Reply:
No reason, I got too lost in the css sauce. Thank you for pointing that out!

These updates plus the calc mishap are fixed here: 2303facb98cd03322324a4c0273528c94aadba93